popSimManu.tex

```diff
@@ -835,6 +835,16 @@ \subsection*{Model quality control}
 Further details of our QC process can be found in our developer documentation
 (\url{https://stdpopsim.readthedocs.io/en/latest/development.html#}).
 
+The possibility for error and the importance of careful qualty control
+was inadvertantly illustrated during our own development process:
+during the final revisions of this paper, we noticed that the 
+OutOfAfrica\_3G09 model \citep{gutenkunst2009inferring}
+had not in fact gone through the QC process.
+Then, the QC process revealed that not only was our implementation slightly wrong --
```

Comment:
I think an em-dash without adjacent spaces is conventionally used in this type of phrasing, rather than an en-dash. I.e. `wrong---migration`.